ttnn/cpp/ttnn/tensor/tensor_ops.hpp

```diff
@@ -0,0 +1,46 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <vector>
```

Comment:
You don't even need this header here :-)

Reply:
Fixed :)